organization/_posts/2023/2023-06-08-coordination.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "HSF Coordination Meeting #251, 8 June 2023"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Kyle Knoepfel, Benedikt Hegner, Pere Mato, Alex Held, Mason Proffitt, Nicole Skidmore, Claire Antel, Patrick Gartung, Uwe Hernandez, Vangelis Kourlitis, Krzysztof Genser, Nick Smith, Michel Jouvin, Wouter Deconick, Stefan Roiser
+
+Apologies/Contributing: Eduardo Rodrigues, Efe Yazgan
+
+## News, general matters, announcements
+
+### LHCP Presentation
+
+Vangelis Kourlitis presented a [talk](https://indico.cern.ch/event/1198609/contributions/5370078/) on *HL-LHC and Beyond Computing Challenges*.
+
+Questions raised regarding:
+
+- ATLAS - Google Cloud Platform project and future
+- Common fast-simulation software à la ACTS
+- CO2 emissions of our disks usage
+
+- *Discussion:* disk energy consumption is much less than CPU. See [A holistic study of the WLCG energy needs for the LHC scientific program](https://indico.jlab.org/event/459/contributions/11499/attachments/9236/14205/WLCGEnergyNeedsCHEP2023.pdf) talk at CHEP2023 (slide 9).
+
+### LHCC
+
+Alessandra Forti, one of the Analysis Facilities Forum co-conveners, presented the status of Analysis Facilities in the community ([slides](https://indico.cern.ch/event/1225017/attachments/2552956/4612708/20230605_LHCC_AF_progress_report-8.pdf) attached).
+
+There was a very productive discussion between the referees, the experiments, WLCG and HSF at the special session.
+
+LHCC thanked the Analysis Facilities Forum for their work bringing the R&D around analysis facilities into the light.
+
+In the whole community, we should take care to balance the R&D with production activities at the sites. Modernising infrastructure, helped by this type of R&D, is already a valuable outcome.
+
+To demonstrate the new analysis facility concepts work, they need to be tested on representative used cases *at scale*. LHCC asks the experiments to provide a document on analysis use cases, which can be matched to AFs, then using this to develop benchmarks.
+
+- The exact parameters of this document will come later, in consultation with the LHCC's HL-LHC review committee.
+
+*Long discussion on this point, we conclude that the LHCC are positive about the R&D and would like to get more formal engagement from the experiments, folding these studies into HL-LHC planning. We think the "concerns" are not new, but this actually means that we are investigating the right topics and sending the right messages.*
```

Comment:
from the text itself it is unclear what "concerns" refers to. Maybe change the wording